### ISSUE TITLE
Remove `DefaultTextStyle` as parent in `README.md` and `example`

### ISSUE THREAD

**Issue by SirusCodes** [aagarwal1012/Animated-Text-Kit#221]
I would suggest to remove use of `DefaultTextStyle` in `README.md` and `example` as it would suggest the user to use it for every time when they want to change the text style

**Comment by awhitford** [comment]
Using `DefaultTextStyle` avoids having to repeat having to specify the `textStyle` for each `AnimatedText` instance.  It cut a lot of lines of code in the sample code.

Having said that, I support the idea of a _section_ that explains the benefits of using `DefaultTextStyle`.

**Comment by awhitford** [comment]
> ... which was introduced in #221 as it would suggest the user to use it for every time when they want to change the text style.

@SirusCodes I think you meant to reference #217.

**Comment by SirusCodes** [comment]
> Using `DefaultTextStyle` avoids having to repeat having to specify the `textStyle` for each `AnimatedText` instance.  It cut a lot of lines of code in the sample code.
>
> Having said that, I support the idea of a _section_ that explains the benefits of using `DefaultTextStyle`.

Yes because if I'm a beginner in Flutter and say this in a package I would think that this is the only way to use this package...

I think we can add a section explaining the use of `DefaultTextStyle` and in our example use it somewhere where we have repetation not in all animation?

**Comment by Nidhi-2000** [comment]
Hey , @SirusCodes can you please assign this issue to me . I am a GSSOC'21 participant

**Comment by Nidhi-2000** [comment]
thanks @Siruscodes.

**Comment by harshita2319** [comment]
I'd like to work on this (GSSOC participant)

**Comment by SirusCodes** [comment]
@harshita2319 go ahead.

**Comment by harshita2319** [comment]
> @harshita2319 go ahead.
> 

Should I add  new file for explaining the use as you've mentioned earlier?

**Comment by SirusCodes** [comment]
Just update the README @harshita2319
